### ISSUE TITLE
Why do testnet.py test results (AP,AP50,AP75,APm,APs,APl) all go to 0?  

### ISSUE THREAD

**Issue by 906096481** [krumo/Domain-Adaptive-Faster-RCNN-PyTorch#38]
For example ('AP',2.2241e-05)

**Comment by krumo** [comment]
Hi, please consider to provide more details about your experimental setting (e.g. your PyTorch version, your cuda version, the task you are working on and so on). If possible, you are welcome to provide your training log. Only with more details could I give you more help.

**Comment by 906096481** [comment]
Hi,This is my training log.The environment is configured as required on GitHub




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "krumo/Domain-Adaptive-Faster-RCNN-PyTorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年4月18日(星期一) 晚上6:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [krumo/Domain-Adaptive-Faster-RCNN-PyTorch] Why do testnet.py test results (AP,AP50,AP75,APm,APs,APl) all go to 0?   (Issue #38)





 
Hi, please consider to provide more details about your experimental setting (e.g. your PyTorch version, your cuda version, the task you are working on and so on). If possible, you are welcome to provide your training log. Only with more details could I give you more help.

**Comment by krumo** [comment]
Unfortunately I cannot find where you place your training log. Could you upload it again?

**Comment by 906096481** [comment]
[log.txt](https://github.com/krumo/Domain-Adaptive-Faster-RCNN-PyTorch/files/8512576/log.txt)

**Comment by 906096481** [comment]
I realized that maybe the weights were not loaded during the test, so the AP value tended to be 0, but I did write it in.

**Comment by krumo** [comment]
In the training log, we could observe that after training the final model is tested on Foggy Cityscapes val and a performance of ~39.8 mAP@50 is obtained. Did you follow the README.md to run 
`python tools/test_net.py --config-file "configs/da_faster_rcnn/e2e_da_faster_rcnn_R_50_C4_cityscapes_to_foggy_cityscapes.yaml" MODEL.WEIGHT <path_to_store_weight>/model_final.pth` to test your final model?

**Comment by 906096481** [comment]
Yes, I followed the tutorial.  I know the training results are fine, but the test results don't know why.

**Comment by 906096481** [comment]
I tried writing the weight path directly to default.py, and the result was the same.

**Comment by krumo** [comment]
It seems the training process is okay. Could you also upload your test log here?

**Comment by 906096481** [comment]
![QTKMFMTZXZ_JKIH52 QPO J](https://user-images.githubusercontent.com/72055357/164000339-91d023ff-9178-4acf-bebe-129a4f8ad1c1.png)

**Comment by puliang292** [comment]
I have the same problem,maybe we can contact each other  to solve the problem in a convenient way such as wechat.

**Comment by 906096481** [comment]
Ok, is it convenient to provide your wechat?

**Comment by krumo** [comment]
> ![QTKMFMTZXZ_JKIH52 QPO J](https://user-images.githubusercontent.com/72055357/164000339-91d023ff-9178-4acf-bebe-129a4f8ad1c1.png)

This screenshot doesn't provide any meaningful information. Please make sure you load the correct checkpoint when performing tests.

**Comment by 906096481** [comment]
> 
Ok, is it convenient to provide your wechat?

**Comment by 906096481** [comment]
> > ![QTKMFMTZXZ_JKIH52 QPO J](https://user-images.githubusercontent.com/72055357/164000339-91d023ff-9178-4acf-bebe-129a4f8ad1c1.png)
> 
> This screenshot doesn't provide any meaningful information. Please make sure you load the correct checkpoint when performing tests.

![1650504901(1)](https://user-images.githubusercontent.com/72055357/164354138-0a0f68e8-1c16-4953-b987-73bf53069485.png)
I have filled in the weight address here, is it correct?

**Comment by 906096481** [comment]
> I have the same problem,maybe we can contact each other to solve the problem in a convenient way such as wechat.

ok.

**Comment by puliang292** [comment]
> > I have the same problem,maybe we can contact each other to solve the problem in a convenient way such as wechat.
> 
> ok.

wechat number:18205202287

**Comment by 906096481** [comment]
> 

OK

**Comment by 906096481** [comment]
> > > I have the same problem,maybe we can contact each other to solve the problem in a convenient way such as wechat.
> > 
> > 
> > ok.
> 
> wechat number:18205202287

OK

**Comment by zhengying455** [comment]
> > > > I have the same problem,maybe we can contact each other to solve the problem in a convenient way such as wechat.
> > > 
> > > 
> > > ok.
> > 
> > 
> > wechat number:18205202287
> 
> OK

打扰您了，我最近在学习这个方向相关的知识，在复现代码的是时候遇到了一些问题，或许可以加一下您的微信请教一下吗，如果可以的话，非常感谢

**Comment by zhengying455** [comment]
> > > I have the same problem,maybe we can contact each other to solve the problem in a convenient way such as wechat.
> > 
> > 
> > ok.
> 
> wechat number:18205202287

打扰您了，我最近在学习这个方向相关的知识，在复现代码的是时候遇到了一些问题，或许可以加一下您的微信请教一下吗，如果可以的话，非常感谢
